### DISCUSSION TITLE
Secure sync of files with OneDrive?

### DISCUSSION THREAD

**Discussion by kabokabeso** [remotely-save/remotely-save#842] [General]
Hello,
Microsoft says that it uses a secure 256-Bit encyption while uploading or downloading files to the drive, so just for the transfer. Is this also the case with remotely safe, so is the transfer encrypted and safe without using the full file encryption in remotely save via an additional passwort? So basically the question is, is using remotely save save in the same way as just uploading files via the onedrive app or via browser to one drive?

**Comment by fyears** [comment]
i don't know what do you mean "256 bit encryption " , and do you have any source link?

the plugin uses official api from microsoft
